user-guide/Getting_started/Accessing_DataMiner/Accessing_DataMiner_Cube/Opening_DataMiner_Cube.md

```diff
@@ -142,6 +190,8 @@ You can pass a number of options when starting DataMiner Cube:
 
 - [buffer=](#buffer)
 
+- [bypassProxy=true](#bypassproxytrue)
```

Comment:
The `bypassProxy` argument is not an option for Cube, it is interpreted by the IIS `dataminer-failover-reverseproxy-rule` rewrite rule. It is not intended to be used directly by end-users and as such I would exclude it from documentation.